src/plugins/presentation_util/public/components/controls/control_group/control_group.scss

```diff
@@ -0,0 +1,186 @@
+$smallControl: $euiSize * 14;
+$mediumControl: $euiSize * 25;
+$largeControl: $euiSize * 50;
+$controlMinWidth: $euiSize * 14;
+
+.controlGroup {
+  margin-left: $euiSizeXS;
+  overflow-x: clip; // sometimes when using auto width, removing a control can cause a horizontal scrollbar to appear.
+  min-height: $euiSize * 4;
+  padding: $euiSize 0;
+}
+
+.controlFrame--cloneWrapper {
+  width: max-content;
+
+  .euiFormLabel {
+    padding-bottom: $euiSizeXS;
+  }
+
+  &-small {
+    width: $smallControl;
+  }
+
+  &-medium {
+    width: $mediumControl;
+  }
+
+  &-large {
+    width: $largeControl;
+  }
+
+  &-twoLine {
+    margin-top: -$euiSize * 1.25;
+  }
+
+  .euiFormLabel, div {
+    cursor: grabbing !important; // prevents cursor flickering while dragging the clone
+  }
+
+  .controlFrame--draggable {
+    cursor: grabbing;
+    height: $euiButtonHeight;
+    align-items: center;
+    border-radius: 6px;
+    @include euiFontSizeS;
+    font-weight: $euiFontWeightSemiBold;
+    box-shadow:
+      #{$euiFormControlBoxShadow},
```

Comment:
@andreadelrio and I chatted over Slack. It might more readable and maintainable if we just import shadows  from EUI and then override the color, for example.